src/pages/Notes/FilterModal/FilterModal.jsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import "./FilterModal.css";
+import { useLabels } from "../../../context/labels-context";
+
+function FilterModal({ filterDispatch, filterState, setOpenFilterModal }) {
+  const { labels } = useLabels();
+
+  return (
+    <div className={`playlist-dropdown-container`}>
```

Comment:
The naming could be better, didn't get the use of playlist name

Reply:
Actually it was prebuild class